plugins/woocommerce-blocks/tests/e2e/tests/product-collection/product-collection.page.ts

```diff
@@ -798,6 +804,14 @@ class ProductCollectionPage {
 		}
 	}
 
+	async removeAdvancedPreview() {
+		await this.page.evaluate( () => {
+			if ( window.__removePreview ) {
```

Comment:
In this case, we should not validate the existence of `__removePreview` like that because we want the test to fail if it's unavailable as early as possible. This makes debugging easier through faster and more accurate error reporting.

Reply:
Addressed as part of: https://github.com/woocommerce/woocommerce/pull/51653/commits/0a7a5d84f1f23e9464b0bcda816bfad22326b486